components/sponsored_benefits/app/controllers/sponsored_benefits/organizations/plan_design_proposals_controller.rb

```diff
@@ -4,12 +4,24 @@ class Organizations::PlanDesignProposalsController < ApplicationController
     include DataTablesAdapter
 
     before_action :load_plan_design_organization
-    before_action :load_plan_design_proposal, only: [:edit, :update, :destroy]
+    before_action :load_plan_design_proposal, only: [:edit, :update, :destroy, :publish_quote]
 
     def index
       @datatable = effective_datatable
     end
 
+    def publish_quote
+      #FIXME, any specific conditions to be verified before publishing?
+      if @plan_design_proposal.may_publish?
+        @plan_design_proposal.publish!
+        flash[:notice] = "Quote Published"
+      else
+        flash[:error] = "Quote failed to publish.".html_safe
+        # FIXME: where to route if error occurs?
+        redirect_to benefit_sponsorship_plan_design_proposals_path(plan_design_proposal.benefit_sponsorship)
```

Comment:
this doesn't look right. I think you'd want:

organizations_plan_design_organization_plan_design_proposals_path(@plan_design_organization)

to just bring them back to the datatable with all that org's proposals listed.